app/routines/search_users.rb

```diff
@@ -39,32 +43,37 @@ class SearchUsers
   def exec(query, options={})
```

Comment:
Yes, if I use standard_read I can leave the search_users alone and not overly complicate it